tests/storage/sdm_indirection_test.py

```diff
@@ -843,10 +845,29 @@ def test_property(self, prop, val):
     def test_functions(self, fn, nargs):
         self.checker.check_method_call(fn, nargs)
 
+    def test_newmetadata(self):
+        args = (
+            1,             # metaId
+            uuid.uuid4(),  # sdUUID
+            uuid.uuid4(),  # imgUUID
+            uuid.uuid4(),  # puuid
+            1000,          # capacity
+            1,             # format
+            2,             # type
+            sc.LEAF_VOL,   # voltype
+            'file',        # disktype
+            '',            # description
+            sc.LEGAL_VOL,
+        )
+        kwargs = {"sequence": 0}
+        self.checker.check_classmethod_call_args_kwargs(
+            "newMetadata",
+            *args,
+            **kwargs)
```

Comment:
I would drop this test since it requires too much effort to maintain and
provide very low value, but since you invested the time in adding working
test we can keep it.

Reply:
yeah, I don't understand what these tests check and what "indirection" means